src/Lykke.blue.Service.ReferralLinks/Models/ModelConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Lykke.blue.Service.ReferralLinks.Core.Assets;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Lykke.blue.Service.ReferralLinks.Models
+{
+    public static class ModelConverter
+    {
+        public static Asset ConvertToServiceModel(this Lykke.Service.Assets.Client.Models.Asset src)
```

Comment:
This code is not using anywhere. Remove this file.

Reply:
Done

Reply:
How it can be done if file still exists?

Reply:
For some reason git didn't pick up as Delete when you remove a file from Visual Studio. Needed to go through all deletes and make sure they are committed. Should be gone now.